src/components/interfaces/MOBILE_API.xml

```diff
@@ -2645,6 +2645,7 @@
         <element name="GetAppServiceDataID" value="53" hexvalue="35" since="5.1" />
         <element name="GetFileID" value="54" hexvalue="36" since="5.1" />
         <element name="PerformAppServiceInteractionID" value="55" hexvalue="37" since="5.1" />
+        <element name="CloseApplicationID" value="56" hexvalue="38" since="5.2" />
```

Comment:
This function id is going to conflict with other functions ids. [](uhttps://github.com/smartdevicelink/rpc_spec/pull/171/filesrl)